packages/material-ui/src/SnackbarContent/SnackbarContent.test.js

```diff
@@ -31,34 +31,38 @@ describe('<SnackbarContent />', () => {
   describe('prop: action', () => {
     it('should render the action', () => {
       const action = <span>action</span>;
-      const wrapper = shallow(<SnackbarContent message="message" action={action} />);
-      assert.strictEqual(wrapper.childAt(1).hasClass(classes.action), true);
-      assert.strictEqual(wrapper.childAt(1).contains(action), true);
+      const { container, getByText } = render(
+        <SnackbarContent message="message" action={action} />,
+      );
+      expect(container.querySelector(`.${classes.action}`)).to.have.class(classes.action);
+      expect(getByText('action')).to.have.property('nodeName', 'SPAN');
```

Comment:
This is basically just repeating what is already put into `action`.

I think it makes more sense to reduce it to a single assertion that the element `.${classes.action}` contains the `action`. You can add a `data-testid="action"` to the element in `action` and then check if this element is contained in `.${classes.action}` by using  https://www.chaijs.com/plugins/chai-dom/#containselector-or-element